### PR TITLE
Publish jar to GH packages

### PR DESCRIPTION
**Pull request by maheshm**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Publish Package
+
+on:
+  push:
+    branches:
+      - develop
+
+env:
```
Comment (maheshm) [inline .github/workflows/publish.yml:8]: Are these still required even if we are not building STK?

Comment (carise) [reply .github/workflows/publish.yml:8]: `ADAM_STK_LICENSE_DATA`, I think, is used as part of the unit tests. The license utility looks in the environment variables for the license contents. You can try removing the env variable here and see if the GH actions pass.

The `SERVER_PROPERTIES_PATH` doesn't seem to be used anywhere, so you can remove it.

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -12,32 +12,19 @@ jobs:
     runs-on: ubuntu-latest
```
Comment (carise) [inline .github/workflows/build-and-test.yml:12]: I couldn't comment on the hidden lines, but I think you might also be able to remove `STK_JARS_PATH` and `STK_PROPERTIES_PATH` here too.

Comment (maheshm) [reply .github/workflows/build-and-test.yml:10]: Yeah, both are not required. But LICENSE_DATA is required

File: .github/workflows/publish.yml (hunk 1 above)
Comment (maheshm) [reply .github/workflows/publish.yml:8]: ADAM_STK_LICENSE_DATA is required. SERVER_PROPERTIES_PATH is not required.

**Review by carise — APPROVED** [review]

**Comment by carise** [comment]
LGTM, pending @HankG 's review.

**Review by HankG — APPROVED** [review]

**Pull request merged**